examples/user_guide/15-Streaming_Data.ipynb

```diff
@@ -483,68 +536,50 @@
-    "    areas = hv.Dataset(data).to(hv.Area, 'index', 'Usage')\n",
-    "    return hv.Area.stack(areas.overlay()).relabel('Memory')\n",
-    "\n",
-    "def cpu_box(data):\n",
-    "    return hv.BoxWhisker(data, 'CPU', 'Utilization').relabel('CPU Usage')\n",
-    "\n",
-    "\n",
-    "####################################################\n",
-    "# Set up StreamingDataFrame and add async callback #\n",
-    "####################################################\n",
-    "\n",
-    "cpu_stream = DataFrameStream(get_cpu_data(), 800)\n",
-    "mem_stream = DataFrameStream(get_mem_data())\n",
+    "buffer = Buffer(np.zeros((0, 2)))\n",
     "\n",
     "@gen.coroutine\n",
     "def f():\n",
-    "    for i in range(500):\n",
-    "        yield gen.sleep(0.01)\n",
-    "        yield cpu_stream.send(get_cpu_data())\n",
-    "        yield mem_stream.send(get_mem_data())\n",
-    "\n",
+    "    for x in range(500):\n",
+    "        gen.sleep(0.1)\n",
+    "        buffer.send(np.array([[x, np.random.rand()]]))\n",
+    "        \n",
     "IOLoop.current().add_callback(f)\n",
+    "hv.DynamicMap(hv.Curve, streams=[buffer]).redim.range(y=(0, 1))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img class=\"gif\" src=\"http://assets.holoviews.org/gifs/guides/user_guide/Streaming_Data/streamz2.gif\"></img>\n",
     "\n",
-    "#######################################\n",
-    "# Define DynamicMaps and display plot #\n",
-    "#######################################\n",
-    "\n",
-    "cpu_dmap = hv.DynamicMap(cpu_box, streams=[cpu_stream])\n",
-    "mem_dmap = hv.DynamicMap(mem_stack, streams=[mem_stream])\n",
-    "\n",
-    "opts = {'plot': dict(width=500, height=400, color_index='CPU'),\n",
-    "        'style': dict(box_fill_color=hv.Cycle('Category20'))}\n",
-    "\n",
-    "(cpu_dmap.redim.range(Utilization=(0, 100)).opts(**opts) +\n",
-    " mem_dmap.redim.range(Usage=(0, 100)).opts(plot=dict(height=400, width=400)))"
+    "Here the plot should update 100 times as before, but now via the Tornado IOLoop which will not block other interactions and work in the notebook."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Real examples"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Using the ``Pipe`` and ``Buffer`` streams we can create complex streaming plots very easily. In addition to the toy examples we presented in this guide it is worth looking at looking at some of the examples using real, live, streaming data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* The [streaming_psutil](http://holoviews.org/gallery/apps/bokeh/stream_psutil.html) bokeh app is one such example which display CPU and memory information using the ``psutil`` library  (install with ``pip install psutil`` or ``conda install psutil``)"
    ]
```

Comment:
You can't put the full gif here, to make it very concrete?

Reply:
I've put it below. I'm still working on another example in which case I'll add both inline here.